ga4gh/datamodel/__init__.py

```diff
@@ -381,18 +381,6 @@ def sanitizeAlignmentFileFetch(
         return referenceName, start, end
 
     @classmethod
-    def sanitizeFastaFileFetch(cls, start=None, end=None):
```

Comment:
There didn't seem to be much point to it --- ``checkQueryRange`` makes sure the values make sense and are in the correct range, so we'd just be doing the same thing again with the sanitize call. The values are already guaranteed to be ints, and they're in the correct range, so we're as safe as we're going to get.